### ISSUE TITLE
Disable automatic table prettifier?

### ISSUE THREAD

**Issue by montoner0** [marktext/marktext#1501]
### Is there a way to disable automatic table prettifier?

Although this feature is nice but sometimes I'd prefer to not prettify tables.

**Comment by fxha** [comment]
Thanks @montoner0  for reporting the issue. Do you mean to use the original table style or rather to not adjust all cells to the largest cell (just insert space before and after the cells text).

**Comment by montoner0** [comment]
I mean the latter (width adjusting). But I'd prefer to have a control over a table style too.

**Comment by Jocs** [comment]
>But I'd prefer to have a control over a table style too.

What do yo mean `control over a table style`?

**Comment by montoner0** [comment]
As far as I know there are several possible table kinds in markdown. It mainly depends on the implementation. At least pandoc offers 4 different ways to make a table. https://pandoc.org/MANUAL.html#tables

**Comment by fxha** [comment]
> As far as I know there are several possible table kinds in markdown. It mainly depends on the implementation.

That's true and there is some issue opened about pandoc and it's additional extensions. We currently only support CommonMark, GFM and some additional features/extensions.

**Comment by jnm** [comment]
> to not adjust all cells to the largest cell (just insert space before and after the cells text)

This would be wonderful :heart:

**Comment by Jocs** [comment]
Thank you for the suggestion. The automatic table prettifier (cell-width padding) is still present in the new editor engine and remains unconditional — there is currently no option to disable it.

This is a valid quality-of-life request for users who prefer minimal diffs in version control, but after six years the feature has seen very little community traction and has not been scheduled for implementation. We are closing this issue as part of a stale-issue sweep.

If there is renewed interest, please open a fresh issue describing the desired behavior (e.g. a "minimal table formatting" preference that emits only a single space around each cell). Community upvotes on a new issue will help prioritize it. Thank you for your patience.
